docs/label-spec.md

```diff
@@ -1,6 +1,6 @@
 # Label Specification
 
-A _label_ (`label.toml`) contains metadata about a box. It is in TOML format, and is considered immutable once stored.
+A _label_ contains metadata about a parcel. It is in TOML format and exists only in the `parcel` array of an invoice
 
 ```toml
 sha256 = 5b992e90b71d5fadab3cd3777230ef370df75f5b...
```

Comment:
Oops.. I see that this line is not well-formed TOML. Definitely not necessary to fix as part of this PR. I just noticed it because of GitHub's syntax highlighting